airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/filters.py

```diff
@@ -51,5 +51,43 @@ def hash(value, hash_type="md5", salt=None):
     return computed_hash
 
 
-_filters_list = [hash]
+def base64encode(value: str) -> str:
+    """
+    Implementation of a custom Jinja2 base64encode filter
+
+    For example:
+
+      OAuthAuthenticator:
+        $ref: "#/definitions/OAuthAuthenticator"
+        $parameters:
+          name: "client_id"
+          value: "{{ config['client_id'] | base64encode }}"
+
+    :param value: value to be encoded in base64
+    :return: base64 encoded string
+    """
+
+    return base64.b64encode(value.encode("utf-8")).decode()
+
+
+def base64decode(value: str) -> str:
```

Comment:
is this a duplicate?

Reply:
No, I've added two filters `base64decode` and `base64encode`